include/itkPhaseCorrelationImageRegistrationMethod.h

```diff
@@ -176,13 +176,13 @@ class ITK_TEMPLATE_EXPORT PhaseCorrelationImageRegistrationMethod: public Proces
 
   /** Set/Get the Operator. */
   itkSetObjectMacro( Operator, OperatorType );
-  itkGetObjectMacro( Operator, OperatorType );
+  itkGetModifiableObjectMacro( Operator, OperatorType );
```

Comment:
We can use `itkGetConstObjectMacro` here unless we find out that we really need `itkGetModifiableObjectMacro`.